tests/test_api.py

```diff
@@ -269,6 +269,30 @@ def test_rms(x, axis, keepdims, expected):
 @pytest.mark.parametrize(
     'duration, sampling_rate, expected',
     [
+        (None, None, np.NaN),
+        (None, 1000, np.NaN),
+        ('', None, np.NaN),
+        ('', 1000, np.NaN),
+        ('none', None, np.NaN),
+        ('none', 1000, np.NaN),
+        (np.NaN, None, np.NaN),
+        (np.NaN, 1000, np.NaN),
+        (pd.NaT, None, np.NaN),
+        (pd.NaT, 1000, np.NaN),
+        (np.timedelta64('NaT', 's'), None, np.NaN),
+        (np.timedelta64('NaT', 's'), 1000, np.NaN),
+        (pd.to_timedelta(np.NaN, 's'), None, np.NaN),
+        (pd.to_timedelta(np.NaN, 's'), 1000, np.NaN),
+        (pd.to_timedelta(pd.NaT, 's'), None, np.NaN),
```

Comment:
I guess we can skip the `to_timedelta()` here.

Reply:
You are right, it's the same output:

```python
>>> pd.NaT
NaT
>>> pd.to_timedelta(pd.NaT, 's')
NaT
>>> pd.to_timedelta(np.NaN, 's')
NaT
```
I removed the tests.